### DISCUSSION TITLE
Detect pressing `.` key

### DISCUSSION THREAD

**Discussion by Im-Oab** [17cupsofcoffee/tetra#280] [Q&A] [answered]
I try to detect pressing `.` `(dot)` key on the keyboard but I can't find `Key::Dot` on `tetra::input::Key`. Does it have another name on `tetra::input::Key`?

**Comment by 17cupsofcoffee** [comment, marked as answer]
It's `Key::Period` - which, in retrospect, I'm not a huge fan of (nobody calls it a period in the UK!), but I think I just copied the name from SDL 😄 

If I get chance, I think it'd be a good idea for me to add doc comments to the `Key` variants that include the actual character, to make Ctrl+F a bit more useful on that doc page.
